Makefile

```diff
@@ -109,6 +110,7 @@ version:
 	@scripts/check_version.sh
 
 docker-thirdparty:
+# 	docker pull couchdb:2.3
```

Comment:
is this correct?

Reply:
We were using 2.2 up until now. I added the above line (commented) so that we move to 2.3 in the near future (which is what Fabric is currently using). I didn't want to make too many changes in this PR.